libs/blocks/pdf-viewer/pdf-viewer.js

```diff
@@ -2,7 +2,7 @@
 
 import { createTag, getConfig, loadScript } from '../../utils/utils.js';
 
-const API_SOURCE_URL = 'https://documentcloud.adobe.com/view-sdk/viewer.js';
+const API_SOURCE_URL = 'https://acrobatservices.adobe.com/view-sdk/viewer.js';
```

Comment:
What about the existing URLs?

Reply:
pdf embed api team has deprecated the old domain. I am working with product team for some other issues related to pdf-viewer and this came up. Hence updated the url. They are still supporting the old domain but will be asking all users to move to new url and completely stop supporting the old one.

Reply:
Right... maybe put different...

"What about existing live content that use the old URL?"

Do we have pages in the wild? Test automation scripts?

Have you talked to stock.adobe.com, business.adobe.com, etc. to ensure they are not using these links and they will stop working when we merge this PR?

Reply:
Sorry didn't got your question first time.
I did talked to bacom team before updating the url. Also consuming teams don't define the api url at their end to use pdf-viewer. They just set up the credentials.  Hence nothing will break for milo consuming teams using pdf-viewer block post this update. Existing pages will not be impacted, it can be seen with after urls in PR description.

Even if consuming teams use the old url for anything at their end not related to pdf-viewer it will not be impacted by this change. As this change will only impact pdf-viewer block for which consuming team don't define the api url as mentioned above.

I just verified in the consuming team repo to make sure that the old url is not used anywhere. And don't see it in bacom, stock, dc and nala repos.

Reply:
For reference adding a live page from bacom https://business.adobe.com/resources/reports/assess-and-accelerate-your-content-velocity/thank-you.html
Same page with changes in this PR: https://main--bacom--adobecom.hlx.live/resources/reports/assess-and-accelerate-your-content-velocity/thank-you?milolibs=pdf-viewer--milo--ruchika4